### PR TITLE
Tolerances

### PR DESCRIPTION
**Pull request by AlguacilJ**
“MIO” comments were removed.
Tolerance parameters for minimal volume and area were deffined and updated.

### CODE REVIEW

File: src/GEOUNED/__init__.py [hunk 1]
```diff
@@ -272,6 +272,7 @@ def Start(self):
        if not Options.Facets:
 
           # decompose all solids in elementary solids (convex ones)
+          print(len(MetaList))
```
Comment (shimwell) [inline src/GEOUNED/__init__.py:275]:
```suggestion
```

this looks a bit like a debug print statement, was this left accidently or is it good to print here?

**Comment by shimwell** [comment]
looks like this PR was made against the geouned main branch. I think the repo strategy is PR against dev branch. Would you be able to edit the target branch

**Comment by shimwell** [comment]
Awesome thanks for the base branch change.

**Comment by shimwell** [comment]
@AlguacilJ I think the files have moves around a bit and caused conflicts. might be best to redo this PR as that might be easier than solving the conflicts. Sorry for all the churn

**Comment by AlvaroCubi** [comment]
Please @AlguacilJ could you redo the PR for dev once you have sync your repository with the latest changes? We believe this would be the easiest way.

**Pull request closed without merging**